root/static/scripts/tests/GuessCase.js

```diff
@@ -183,6 +183,36 @@ test('Work', function (t) {
             expected: "Acte 1, no. 7 : Chœur : « voyons brigadier »",
             mode: "French", roman: false, keepuppercase: false
         },
+        {
+            input: "La chenille",
+            expected: "La Chenille",
+            mode: "French", roman: false, keepuppercase: false
+        },
+        {
+            input: "Le téléphone",
+            expected: "Le Téléphone",
+            mode: "French", roman: false, keepuppercase: false
+        },
+        {
+            input: "Les corons",
+            expected: "Les Corons",
+            mode: "French", roman: false, keepuppercase: false
+        },
+        {
+            input: "L'envie",
+            expected: "L'Envie",
+            mode: "French", roman: false, keepuppercase: false
+        },
+        {
+            input: "L’envie",
+            expected: "L’Envie",
+            mode: "French", roman: false, keepuppercase: false
+        },
+        {
+            input: "Les jardins du ciel",
+            expected: "Les jardins du ciel",
```

Comment:
Actually, the correct capitalization for this title would be “Les Jardins du ciel” because it is [_not a verb phrase_](https://musicbrainz.org/doc/Style/Language/French#Summary_of_Rules_for_MusicBrainz).

Please use “La guerre de Troie n’aura pas lieu” instead.